Packs/AttackIQFireDrill/Integrations/AttackIQFireDrill/AttackIQFireDrill.py

```diff
@@ -263,7 +263,7 @@ def get_page_number_and_page_size(args):
 ''' COMMANDS + REQUESTS FUNCTIONS '''
 
 
-def test_module():
+def module_test():
```

Comment:
why did you change the name?

Reply:
The ide sometimes recognizes functions that start with `test_` as test functions that should be ran with pytest, this caused me some odd behaviors while debugging. Either way, I reverted this, as it's not required.